Assignments/Program.cs

```diff
@@ -1,3 +1,42 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T27 - Implementing Stack<T> class
+// ---------------------------------------------------------------------------------------
 
-Console.WriteLine("Hello, World!");
+using System;
+namespace Spark23;
+
+public class T27 {
+   public static void Main () {
+      var stack = new TStack<int> ();
+      stack.Push (1);
+      stack.Peek ();
+      stack.Pop ();
+   }
+   public class TStack<T> {

```

Comment:
No comments for the class/methods/properties.
No #regions too.
Also arrange the properties/methods in alphabetical order.